### PR TITLE
Add pull-ova-all ci prow job for image-builder

### PR DESCRIPTION
**Pull request by kkeshavamurthy**
Adding prow CI job for image-builder OVAs

CC: @codenrhoden @CecileRobertMichon

### CODE REVIEW

File: config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+              path: tls.key
+
+presubmits:
+  kubernetes-sigs/image-builder:
+    - name: pull-ova-all
+      labels:
+        preset-dind-enabled: "true"
+        preset-kind-volume-mounts: "true"
+        preset-cluster-api-provider-vsphere-e2e-config: "true"
+      run_if_changed: 'images/capi/.*'
```
Comment (CecileRobertMichon) [inline config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml:80]: this job should be disabled by default and optional while we test https://github.com/kubernetes-sigs/image-builder/pull/425 with it. Once https://github.com/kubernetes-sigs/image-builder/pull/425 merges, we can update it to run on all PRs that touch capi/

**Review by CecileRobertMichon — CHANGES_REQUESTED** [review]

File: config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml (hunk 1 above)
Comment (CecileRobertMichon) [reply config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml:80]: see https://github.com/kubernetes/test-infra/pull/18532/files#r463098679

Comment (kkeshavamurthy) [reply config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml:80]: makes sense. Thanks.

Comment (CecileRobertMichon) [reply config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml:80]: you need to remove `run_if_changed` and add `optional: true`

**Comment by dims** [comment]
@kkeshavamurthy please fix up the broken tests, see the CI job output.

**Comment by kkeshavamurthy** [comment]
@dims , yes looking into it. Seems like I might be redefining the presets instead of reusing them.

File: config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml (hunk 1 above)
Comment (kkeshavamurthy) [reply config/jobs/kubernetes-sigs/image-builder/image-builder-ova-presubmits.yaml]: Duh. Done.

**Pull request merged**